### PR TITLE
Update ca.m3u

### PR DESCRIPTION
**Pull request by Sphinxroot**
added french channel from quebec canada

### CODE REVIEW

File: channels/ca.m3u [hunk 1]
```diff
@@ -7,28 +7,68 @@ http://158.69.124.9:1935/5aabtv/5aabtv/playlist.m3u8
...
+#EXTINF:-1 tvg-language="French" tvg-logo="https://github.com/Sphinxroot/Iptv-free-direct/blob/master/Images/Logo%20Station%20QC/ICI-Tele.png?raw=true" group-title="ICI Tele Canada  FR(Geo)",ICI Tele Ottawa-Gatineau
+https://rcavlive.akamaized.net/hls/live/664046/cancboft/master_2500.m3u8
+#EXTINF:-1 tvg-language="French" tvg-logo="https://github.com/Sphinxroot/Iptv-free-direct/blob/master/Images/Logo%20Station%20QC/ICI-Tele.png?raw=true" group-title="ICI Tele Canada  FR(Geo)",ICI Tele Quebec
+https://rcavlive.akamaized.net/hls/live/664045/cancbvt/master_2500.m3u8
+#EXTINF:-1 tvg-language="French" tvg-logo="https://github.com/Sphinxroot/Iptv-free-direct/blob/master/Images/Logo%20Station%20QC/ICI-Tele.png?raw=true" group-title="ICI Tele Canada  FR(Geo)",ICI Tele Saguenay
+https://rcavlive.akamaized.net/hls/live/704023/cancktv/master_2500.m3u8
+#EXTINF:-1 tvg-language="French" tvg-logo="https://github.com/Sphinxroot/Iptv-free-direct/blob/master/Images/Logo%20Station%20QC/ICI-Tele.png?raw=true" group-title="ICI Tele Canada  FR(Geo)",ICI Tele Saskatchewan
+https://rcavlive.akamaized.net/hls/live/704019/cancbkft/master_2500.m3u8
```
Comment (boomski) [inline channels/ca.m3u:61]: Geoblocked streams

File: channels/ca.m3u [hunk 2]
```diff
@@ -7,28 +7,68 @@ http://158.69.124.9:1935/5aabtv/5aabtv/playlist.m3u8
...
+https://ici-i.akamaihd.net/hls/live/873426/ICI-Live-Stream/59f8cf257de98a7e0db5ea592e9e081b/1.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="English" tvg-logo="https://i.imgur.com/G7hGk8y.png" group-title="",IIPC TV
 https://uni10rtmp.tulix.tv/iipctv/iipctv.smil/chunklist_w1550531622_b864000.m3u8
 #EXTINF:-1 tvg-id="Knowledge Kids-CA" tvg-name="Knowledge Kids-CA" tvg-language="English" tvg-logo="https://i.imgur.com/kdgMVOI.png" group-title="Kids",Knowledge Kids
 http://knstream1.azureedge.net/knlive/knlive_high.m3u8
+#EXTINF:-1 tvg-language="French" tvg-logo="https://github.com/Sphinxroot/Iptv-free-direct/blob/master/Images/Logo%20Station%20QC/lcn.png?raw=true" group-title="Canadian TV FR",LCN 720
+https://tvalcn.akamaized.net/hls/live/2013847/tvalcn01/low_720p/low_720p.m3u8
+#EXTINF:-1 tvg-language="French" tvg-logo="https://github.com/Sphinxroot/Iptv-free-direct/blob/master/Images/Logo%20Station%20QC/lcn.png?raw=true" group-title="Canadian TV FR",LCN HD
+https://tvalcn.akamaized.net/hls/live/2013847/tvalcn01/high_1080p/high_1080p.m3u8
```
Comment (boomski) [inline channels/ca.m3u:71]: Geoblocked streams

File: channels/ca.m3u [hunk 3]
```diff
@@ -65,3 +107,13 @@ http://toronto3.live247stream.com/toronto360/tv/playlist.m3u8
...
+https://tvalive.akamaized.net/hls/live/2012413/tva01/master.m3u8
+#EXTINF:-1 tvg-language="French" tvg-logo="https://github.com/Sphinxroot/Iptv-free-direct/blob/master/Images/Logo%20Station%20QC/TVA.png?raw=true" group-title="Canadian TV FR",TVA HD
+https://tvalive.akamaized.net/hls/live/2012413/tva01/high_1080p/high_1080p.m3u8
+#EXTINF:-1 tvg-language="French" tvg-logo="https://github.com/Sphinxroot/Iptv-free-direct/blob/master/Images/Logo%20Station%20QC/Yoopa.png?raw=true" group-title="Canadian TV FR",Yoppa 720p Low
+https://tvayoopa.akamaized.net/hls/live/2013885/tvayoopa01/low_720p/low_720p.m3u8
+#EXTINF:-1 tvg-language="French" tvg-logo="https://github.com/Sphinxroot/Iptv-free-direct/blob/master/Images/Logo%20Station%20QC/Yoopa.png?raw=true" group-title="Canadian TV FR",Yoppa 720p Med
+https://tvayoopa.akamaized.net/hls/live/2013885/tvayoopa01/med_720p/med_720p.m3u8
+#EXTINF:-1 tvg-language="French" tvg-logo="https://github.com/Sphinxroot/Iptv-free-direct/blob/master/Images/Logo%20Station%20QC/Yoopa.png?raw=true" group-title="Canadian TV FR",Yoppa HD
+https://tvayoopa.akamaized.net/hls/live/2013885/tvayoopa01/high_1080p/high_1080p.m3u8
```
Comment (boomski) [inline channels/ca.m3u:119]: Geoblocked streams

**Review by boomski — CHANGES_REQUESTED** [review]
Plenty of geoblocked streams  ( only works in Canada)

**Comment by Sphinxroot** [comment]
geoblocked i got many link that does and i use a vpn to stream them

**Review by freearhey — CHANGES_REQUESTED** [review]
Please remove the geo-blocked channels from PR so we can approve it.

**Pull request closed without merging**